version.json

```
@@ -1,3 +1,3 @@
 {
-  "version": "v0.1.1"
+  "version": "v0.1.2"
```

nit: go version bump, so minor bump makes sense
```suggestion
  "version": "v0.2.0"
```